content/applications/finance/fiscal_localizations/united_states.rst

```diff
@@ -429,9 +432,16 @@ Optionally set a :guilabel:`Check Top Margin` and :guilabel:`Check Left Margin`,
 Once all check configurations are complete, :guilabel:`Save` the settings.
 
 .. tip::
-   Some of the check formats may require pre-printed paper from a third party vendor,
+   Pre-printed checks (non-blank check) formats require pre-printed paper from a third party vendor,
```

Comment:
```suggestion
   Pre-printed check formats (non-blank checks) require pre-printed paper from a third-party vendor.
```
I think this just sounds a little clearer- but not a necessary edit. Also, looks like third-party gets a hyphen according to my search and Grammarly (I didn't realize myself!).